### DISCUSSION TITLE
workflow

### DISCUSSION THREAD

**Discussion by robbigoal** [ModelSurge/sd-webui-comfyui#53] [General]
how to get the workflow ?

**Comment by ljleb** [comment]
If you search for "Webui" in comfyui, you should be able to see a node called "WebuiCheckpointLoader". Using this node will allow you to use the currently loaded model in the webui from within comfyui. Is this what you wanted to know?

We are looking to enhance the communication between comfyui and the webui with additonal nodes. You will find our roadmap in the readme.

**Comment by PladsElsker** [comment]
if you want to save the workflow, just hit ctrl+s
